compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -2372,17 +2383,26 @@ setLoweringConfigForComputeOps(mlir::FunctionOpInterface entryPointFn,
       scalableTileFlagsList = rootLoweringConfig.getScalableTileFlagVals();
       if (tilingConfig.getNumTilingLevels() > 0) {
         tileSizesList[tilingConfig.getDistributionLevel()] = distTileSizes;
+        scalableTileFlagsList[tilingConfig.getDistributionLevel()] =
+            distScalableTileSizes;
       }
       if (tilingConfig.getNumTilingLevels() > 1) {
         tileSizesList[tilingConfig.getVectorCommonParallelLevel()] =
             commonVecTileSizes;
+        scalableTileFlagsList[tilingConfig.getVectorCommonParallelLevel()] =
+            commonVecScalableTileFlags;
       }
     } else {
       // Build 4-level lowering configs for other ops.
       tileSizesList = {distTileSizes, commonVecTileSizes};
       SmallVector<int64_t> zeros(numLoops, 0);
+      SmallVector<bool> falseVec(numLoops, 0);
+      // No scalable tiling for the distribution
+      scalableTileFlagsList.push_back(falseVec);
+      scalableTileFlagsList.push_back(commonVecScalableTileFlags);
       TypeSwitch<Operation *>(op)
           .Case<tensor::PackOp>([&](auto packOp) {
+            // TODO: Handle scalable flags
```

Comment:
Should we have an assertion? It sounds very bad if they are failed silently

Reply:
Or maybe we can return failures?

Reply:
I'll add a failure, though it's going to be a bit awkward given that ATM this `TypeSwitch` doesn't return anything :)